exdb/templates/exdb/experience_approval.html

```diff
@@ -11,66 +11,17 @@
         <h1>{% trans "Experience Approval" %}</h1>
         <p><a href="{% url 'hallstaff_dash' %}">Return to Hall Staff Dashboard</a></p>
         <span class="form-errors">{{ form.non_field_errors }}</span>
+        <span class="form-errors">{{ comment_form.non_field_errors }}</span>
```

Comment:
[`non_field_errors` is an iterable](https://docs.djangoproject.com/en/1.9/ref/forms/api/#django.forms.Form.non_field_errors). So you should probably be looping over it.
And since you're looping over both of them, you can use the [`add` template filter](https://docs.djangoproject.com/en/1.9/ref/templates/builtins/#add) to merge the lists and loop over them at the same time.